lib/chef/knife/ec_restore.rb

```diff
@@ -136,10 +136,10 @@ def restore_users
             # Supply password for new user
             user_with_password = user.dup
             user_with_password['password'] = SecureRandom.hex
-            rest.post_rest('users', user_with_password)
+            rest.post('users', user_with_password)
```

Comment:
This (and the user put below) is the only API call here that should actually need v0, but it is good to keep them all consistent. :+1: